components/ILIAS/Authentication/classes/Setup/class.ilSessionMaxIdleIsSetObjective.php

```diff
@@ -136,7 +136,12 @@ public function achieve(Setup\Environment $environment): Setup\Environment
 
     public function isApplicable(Setup\Environment $environment): bool
     {
-        return true;
+        $conn = @fsockopen("localhost", 80);
```

Comment:
Hi @daniwe4 ,

thanks for your PR.

I personally don't like the `@` operator.

I suggest temporarily modifying the error handler and use `try/catch/finnaly` blocks and reset the error handler in the `finally` block.

```php
        set_error_handler(static function (int $severity, string $message, string $file, int $line): never {
            throw new ErrorException($message, $severity, $severity, $file, $line);
        });

        try {
// ...
        } catch (Exception $e) {
// ...
        } finally {
            restore_error_handler();
        }
```

Best regards,
Michael

Reply:
Thank you @mjansenDatabay ,

it's done.